### PR TITLE
DOCSP-33564 - v2.22 updates

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-33564
Staging:
- [Compatibility](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-33564-new-version/compatibility/)
- [What's New](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-33564-new-version/whats-new/#what-s-new-in-2.22)

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [X] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [X] Are all the links working?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -12,12 +12,26 @@ What's New
...
+What's New in 2.22
+------------------
+
+The 2.22 driver release includes the following new features:
+
+- Improvements and bug fixes to the LINQ3 integration.
+- Added support for ``$set`` and ``$vectorSearch`` aggregation stages.
+- Added synonym support for Atlas Search.
+- Reduced memory allocation when using encryption.
+- Added additional logging messages.
```
Comment (ccho-mongodb) [inline source/whats-new.txt:33]: Issue:
The vale action picked up an issue with the term "additional". Please address these or open a new JIRA ticket for the [vale epic](https://jira.mongodb.org/browse/DOCSP-31883) if it is a false positive or if there is some other issue with it.

Suggestion:
I think both "additional" and "more" don't help in understanding the change. I think providing some details or linking to information on what type of logging messages were added would be helpful for users.

File: source/whats-new.txt [hunk 2]
```diff
@@ -12,12 +12,26 @@ What's New
...
+.. _version-2.22:
+
+What's New in 2.22
+------------------
+
+The 2.22 driver release includes the following new features:
+
+- Improvements and bug fixes to the LINQ3 integration.
+- Added support for ``$set`` and ``$vectorSearch`` aggregation stages.
+- Added synonym support for Atlas Search.
```
Comment (ccho-mongodb) [inline source/whats-new.txt:31]: Question:
Are there any docs that can be linked to that describe these features?

**Review by ccho-mongodb — APPROVED** [review]
LGTM. Added a minor issue and question.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**